src/sentry/static/sentry/app/components/dropdownLink.jsx

```diff
@@ -44,6 +44,7 @@ class DropdownLink extends React.Component {
       disabled,
       title,
       caret,
+      caretPadding,
```

Comment:
I would try to avoid using these pretty situational presentational props. In this case, you can avoid doing this completely by styling the `title` in the component that uses `DropdownLink`.

Reply:
Good idea, thank you!